app/functions.php

```diff
@@ -47,28 +47,37 @@ function get_posts(){
 	
 
 	// Return filtered posts
-	return $the_posts;
+	return array_reverse($the_posts);
 
 }
 
 
 
-
-
 /* ###### Submit Lost Pet ###### */
 
-	/* Fields
-	- Type
-	- Breed
-	- Size
-	- Colour
-	- Chipped?
-	- Collar? (colour)
-	- Picture(s)
-	- Location - Post Code/Geo Location
-	- Last Seen Date
-	- Any Other Notes
-	*/
+function submit_lost(){
+	header('Location:http://localhost/');
+	$post_params = $_POST;
+	$post_params['time'] = $_SERVER['REQUEST_TIME'];
+	$post_params['status'] = 'lost';
+
```

Comment:
Might have been because I grabbed some of it from a snippet I had saved, so possibly the c&p caused weird spacing. Have reset it all now